src/rust/Cargo.toml

```diff
@@ -32,10 +32,10 @@ toml = "=0.7.8"
 # N.
 
 [dependencies.soroban-env-host-curr]
-version = "=20.0.1"
+version = "=20.1.0"
 git = "https://github.com/stellar/rs-soroban-env"
 package = "soroban-env-host"
-rev = "2569460880ee16db43b1cd4efb39bc4e5dbb7376"
+rev = "c1b238b65bfd13666be4ac14e0e390c31b549caf"
 
 # This copy of the soroban host is _optional_ and only enabled during protocol
 # transitions. When transitioning from protocol N to N+1, the `curr` copy
```

Comment:
Now that this will be the first supported v20 version of core, what should the revision be for soroban-env-host-prev, @graydon? Maybe it doesn't matter since core will not be built using --enable-protocol-upgrade-via-soroban-env-host-prev flag. 
Or should we put the commit hash of PR just prior to the final commit of env?

Reply:
IIUC that shouldn't matter yet.